ansible/README.md

```diff
@@ -0,0 +1,59 @@
+## Install Ansible
+First you must install ansible on the machine you wish to administer the servers from. This is probably your laptop or other local machine.
+
+For example, this can be done with `sudo apt install ansible` on many linux systems, or `brew install ansible` on mac etc. To check the installation instructions for your machine, check the [ansible installation guide](https://docs.ansible.com/ansible/latest/installation_guide/index.html).
+
+## Create an `inventory` file
+In PerfectFit's cloud deployments we have a `prod` and a `stage` server (for production and staging, respectively). To run any of the ansible playbooks here, you will need to create an `inventory` config file locally, to tell ansible where these servers actually are. Note that this is also called a `hosts` file.
+
+For the current setup, your inventory file will look something like the following:
```

Comment:
Is it possible to have an example inventory file called `invetory_example` or something like that? it would be easier to just change the values in the file

Reply:
Good idea. I have created an `inventory.example` file, and added a note to the README about it.